src/components/LayerUI.tsx

```diff
@@ -381,40 +378,13 @@ const LayerUI = ({
     );
   };
 
-  const dialogs = (
-    <>
-      {appState.isLoading && <LoadingMessage delay={250} />}
-      {appState.errorMessage && (
-        <ErrorDialog
-          message={appState.errorMessage}
-          onClose={() => setAppState({ errorMessage: null })}
-        />
-      )}
-      {appState.showHelpDialog && (
-        <HelpDialog
-          onClose={() => {
-            setAppState({ showHelpDialog: false });
-          }}
-        />
-      )}
-      {appState.pasteDialog.shown && (
-        <PasteChartDialog
-          setAppState={setAppState}
-          appState={appState}
-          onInsertChart={onInsertElements}
-          onClose={() =>
-            setAppState({
-              pasteDialog: { shown: false, data: null },
-            })
-          }
-        />
-      )}
-    </>
-  );
-
   return device.isMobile ? (
     <>
-      {dialogs}
+      <CommonUIDialogs
```

Comment:
any better name ?

Reply:
there's not much commonality so can't say :)

Reply:
maybe split it into `<UIMessages/>`, `<HelpDialog/>`, and `<PasteDialog/>`?